README.md

```diff
@@ -1,4 +1,7 @@
 <p align="center">
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
```

Comment:
In the preview, it does not load the shield... it looks like this :point_down: 
![Screenshot from 2020-08-27 16-30-29](https://user-images.githubusercontent.com/28131369/91496957-ab90af80-e882-11ea-9108-8b11ab1a80ca.png)
And I would prefer it to be next to the other shields